src/main/cpp/src/genomicsdb/query_variants.cc

```diff
@@ -484,9 +489,18 @@ bool VariantQueryProcessor::scan_handle_cell(const VariantQueryConfig& query_con
       for (auto i=0ull; i<num_entries_in_tmp_pq_buffer; ++i)
         end_pq.push(tmp_pq_buffer[i]);
       //Can handle overlapping deletions and reference blocks - if something else, throw error
-      if (!curr_call.contains_deletion_or_MNV() && !curr_call.is_reference_block())
-        throw VariantQueryProcessorException("Unhandled overlapping variants at columns "+std::to_string(curr_call.get_column_begin())+" and "
-                                             + std::to_string(cell.get_begin_column())+" for row "+std::to_string(cell.get_row()));
+      if (!curr_call.contains_deletion_or_MNV() && !curr_call.is_reference_block()) {
+        std::string begin_name, cell_begin_name;
+        int64_t begin_position, cell_begin_position;
```

Comment:
Could we initialize these `int64_t`'s to `-1`? My thought process being if the `assert`'s have been turned off and the calls fail whatever random value is in these variables may be confused for a legitimate position vs. `-1` would be a sentinel that something has gone wrong. Or will these calls never fail in practice?

Reply:
Actually,  the `assert`'s would not be triggered in the `release` at all, so the exception message would not get the coordinates at all.